-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 5, Failures: 3, Errors: 0, Skipped: 0, Time elapsed: 22.395 s <<< FAILURE! - in TestSuite
BankingTestCases1.AddPayeeDatadriven.addPayee  Time elapsed: 1.046 s  <<< FAILURE!
java.lang.AssertionError: expected [The new payee Payee1 was successfully created.] but found [The new payee Payee2 was successfully created.]
	at BankingTestCases1.AddPayeeDatadriven.addPayee(AddPayeeDatadriven.java:84)

BankingTestCases1.AddPayeeDatadriven.addPayee  Time elapsed: 1.059 s  <<< FAILURE!
java.lang.AssertionError: expected [The new payee Payee1 was successfully created.] but found [The new payee Payee3 was successfully created.]
	at BankingTestCases1.AddPayeeDatadriven.addPayee(AddPayeeDatadriven.java:84)

BankingTestCases1.AddPayeeDatadriven.addPayee  Time elapsed: 1.03 s  <<< FAILURE!
java.lang.AssertionError: expected [The new payee Payee1 was successfully created.] but found [The new payee Payee4 was successfully created.]
	at BankingTestCases1.AddPayeeDatadriven.addPayee(AddPayeeDatadriven.java:84)

